### DISCUSSION TITLE
共享刮削数据

### DISCUSSION THREAD

**Discussion by 1525745393** [sqzw-x/mdcx#102] [General]
能不能添加一个功能：刮削到的nfo元数据，转换成群晖Video Station专用vsmeta元数据。共享刮削数据

**Comment by sqzw-x** [comment]
功能请求应该是 issue 而不是 discussion
你可以搜索是否已经有 nfo 转 vsmeta 的工具，此外，如果你希望此功能，那么至少应该提供 vsmeta 的格式规范

**Reply by 1525745393** [reply]
可以参考一下https://github.com/JuanWoo/nfo-to-vsmeta不完善背景和封面写不进去

**Reply by 1525745393** [reply]
此 Video Station API 文档https://download.synology.com/download/Document/Software/DeveloperGuide/Package/VideoStation/All/enu/Synology_Video_Station_API_enu.pdf
